include/native/meson.build

```diff
@@ -0,0 +1,17 @@
+install_subdir(
+  'directx',
+  install_dir: get_option('includedir') / 'dxvk',
+  strip_directory: true,
+)
+
+install_subdir(
+  'windows',
+  install_dir: get_option('includedir') / 'dxvk',
+  strip_directory: true,
+)
+
+install_headers(
+  'wsi/native_wsi.h',
+  'wsi/native_@0@.h'.format(dxvk_wsi),
+  subdir: 'dxvk/wsi',
+)
```

Comment:
This assumes that dependent code would be correct to do things like:

```
#include <d3d9.h>
#include <windows.h>
#include <wsi/native_wsi.h>
```

and then compile with something like

```
$(CC) ... `pkg-config --cflags --libs dxvk-d3d9`
```

which would typically expand to something like `-I/usr/include/dxvk -ldxvk_d3d9`. Is that the intention?

Reply:
Normally it's a matter of opinion whether to install headers directly into `${includedir}` (like e.g. libjpeg and libwayland), or install them into `${includedir}/${package}` and requiring library users to specify an appropriate `-I` option [for parallel-installability](https://ometer.com/parallel.html) (like GTK and vkd3d).

But for DXVK Native specifically, when the headers that are installed include something as namespace-grabbing as `windows.h`, I don't think there's really any alternative to installing into a subdirectory. I arbitrarily chose `dxvk`, but it can be anything (`dxvk-2.x` or `dxvk-native` or even `dxvk/native` would be equally good), as long as it's consistent between here and the `pkg.generate()` calls. It's the equivalent of the `vkd3d` in `/usr/include/vkd3d` or the `gtk-3.0` in `/usr/include/gtk-3.0`.

Reply:
FYI: gitmodules and subprojects are not included in github generated source tarballs.
`directx` directory will be empty in this case.

Reply:
Building DXVK from a github-generated source tarball cannot be supported as long as it needs submodules (this is already true, I'm not changing that here). Currently, it either needs to be built from a recursive git clone, or via some process that vendors the necessary submodule code.

That's probably better discussed on a new issue report rather than here.